### PR TITLE
better debug output

### PR DESCRIPTION
**Pull request by boneskull**
- make sure that template strings aren't used in debug statements, since they are eager

Ref: #4198

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions.

### CODE REVIEW

**Comment by juergba** [comment]
> make sure that template strings aren't used in debug statements, since they are eager

I don't understand this statement.

File: lib/cli/config.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ const parsers = (exports.parsers = {
  */
 exports.loadConfig = filepath => {
   let config = {};
-  debug(`loadConfig: "${filepath}"`);
+  debug('loadConfig: "%s"', filepath);
```
Comment (arvidOtt) [inline lib/cli/config.js:72]: I would suggest to change `line 98` as well to `debug('findConfig: found "%s"', filepath);`

**Comment by boneskull** [comment]
> > make sure that template strings aren't used in debug statements, since they are eager
> 
> I don't understand this statement.

@juergba I mean that a template literal is always evaluated regardless.  If debug is not enabled, the string containing placeholders _won't_ be evaluated.  The impact is probably minimal in our case, but IMO it's good practice.

File: lib/cli/config.js (hunk 1 above)
Comment (boneskull) [reply lib/cli/config.js:72]: done

File: bin/mocha [hunk 2]
```diff
@@ -108,7 +108,11 @@ if (Object.keys(nodeArgs).length) {
     unparse(mochaArgs, {alias: aliases})
   );
```
Comment (juergba) [inline bin/mocha:110]: more line 37

File: lib/cli/config.js [hunk 3]
```diff
@@ -69,7 +69,7 @@ const parsers = (exports.parsers = {
  */
 exports.loadConfig = filepath => {
   let config = {};
-  debug(`loadConfig: "${filepath}"`);
+  debug('loadConfig: found "%s"', filepath);
 
   const ext = path.extname(filepath);
   try {
```
Comment (juergba) [inline lib/cli/config.js:75]: more line 98

File: lib/cli/options.js [hunk 4]
```diff
@@ -178,16 +178,16 @@ const loadPkgRc = (args = {}) => {
     try {
```
Comment (juergba) [inline lib/cli/options.js:178]: more line 85, 88

**Comment by boneskull** [comment]
this is low-impact, so merging

**Pull request merged**